tests/ast/data/df_limit.test

```diff
@@ -0,0 +1,19 @@
+## TEST CASE
+
+df = session.table("test_table")
+
+df = df.select(col("*")).limit(3)
+
+df = df.select(col("*")).limit(n=1, offset=1)
+
+## EXPECTED OUTPUT
+
+res1 = session.table("test_table")
+
+res2 = res1.select("*")
+
+res3 = res2.limit(3, 0)
```

Comment:
Same

Reply:
This is the limit test case so it should be good